docarray/index/backends/hnswlib.py

```diff
@@ -438,6 +438,14 @@ def _get_num_docs_hnsw(self) -> int:
         hnsw_num_docs = first_hnsw_index.element_count
         return hnsw_num_docs
 
+    def _count_docs_in_hnswlib(self) -> int:
+        # Use HNSWlib's method to count documents
+        return self._hnsw_indices.element_count
```

Comment:
remember from last iteration, this is a dictionary, not an index itself, u need to get the first index in the dictionary